configs/groupfree3d/groupfree3d_8x4_sunrgbd-3d-10class-L6-O256.py

```diff
@@ -0,0 +1,176 @@
+_base_ = [
+    '../_base_/datasets/sunrgbd-3d-10class.py',
+    '../_base_/models/groupfree3d.py', '../_base_/schedules/schedule_3x.py',
+    '../_base_/default_runtime.py'
+]
+# model settings
+model = dict(
+    bbox_head=dict(
+        num_classes=10,
+        size_cls_agnostic=True,
+        bbox_coder=dict(
+            type='GroupFree3DBBoxCoder',
+            num_sizes=10,
+            num_dir_bins=12,
+            with_rot=True,
+            size_cls_agnostic=True,
+            axis_aligned_lw=True,
+            mean_sizes=[[2.114256, 1.620300, 0.927272],
+                        [0.791118, 1.279516, 0.718182],
+                        [0.923508, 1.867419, 0.845495],
+                        [0.591958, 0.552978, 0.827272],
+                        [0.699104, 0.454178, 0.75625],
+                        [0.69519, 1.346299, 0.736364],
+                        [0.528526, 1.002642, 1.172878],
+                        [0.500618, 0.632163, 0.683424],
+                        [0.404671, 1.071108, 1.688889],
+                        [0.76584, 1.398258, 0.472728]]),
+        sampling_objectness_loss=dict(
+            type='FocalLoss',
+            use_sigmoid=True,
+            gamma=2.0,
+            alpha=0.25,
+            loss_weight=2.0),
+        objectness_loss=dict(
+            type='FocalLoss',
+            use_sigmoid=True,
+            gamma=2.0,
+            alpha=0.25,
+            loss_weight=4.0),
+        center_loss=dict(
+            type='SmoothL1Loss',
+            beta=1.0 / 9.0,
+            reduction='sum',
+            loss_weight=10.0),
+        dir_class_loss=dict(
+            type='CrossEntropyLoss', reduction='sum', loss_weight=1.0),
+        dir_res_loss=dict(
+            type='SmoothL1Loss',
+            beta=0.04,
+            reduction='sum',
+            loss_weight=10.0 * 0.04),
+        size_reg_loss=dict(
+            type='SmoothL1Loss',
+            beta=0.0625,
+            reduction='sum',
+            loss_weight=10.0 * 0.0625),
+        semantic_loss=dict(
+            type='CrossEntropyLoss', reduction='sum', loss_weight=1.0)),
+    test_cfg=dict(
+        sample_mod='kps',
+        nms_thr=0.25,
+        score_thr=0.0,
+        per_class_proposal=True,
+        prediction_stages='all'))
+
+dataset_type = 'SUNRGBDDataset'
```

Comment:
Should delete the shared parts with base sunrgbd config and change `shift_height`